pkg/agent/cniserver/pod_configuration.go

```diff
@@ -66,6 +66,11 @@ func setupInterface(podName string, podNamespace string, ifname string, netns ns
 		containerIface.Sandbox = netns.Path()
 		hostIface.Name = hostVeth.Name
 		hostIface.Mac = hostVeth.HardwareAddr.String()
+		// TODO: should be conditional on the OVS datapath type
+		// Waiting for https://github.com/vmware-tanzu/antrea/pull/125 to be merged
```

Comment:
waiting for https://github.com/vmware-tanzu/antrea/pull/125 to be merged and I will update this PR